### PR TITLE
Adding maximum panes limit

### LINKED ISSUE #305: Limit the number of panes in a window
**What's missing? Please describe your flow and where this feature will help.**

When I'm dmeoing the appsome people get cute and split the windows till it breaks. A better way will be to add a configuration setting with the maximum number of panes per window.

The default should be 7.

### PR DESCRIPTION
**Pull request by eyal-sasson**
Closes #305.
Also cleaned up divide buttons updates and added config entries for `max_panes` and `min_pane_size` (in %).

### CODE REVIEW

File: src/terminal7.js [hunk 1]
```diff
@@ -154,13 +156,13 @@ export class Terminal7 {
                 .addEventListener("click", () => this.toggleHelp())
         document.getElementById("help-button")
                 .addEventListener("click", () => this.toggleHelp())
-        document.getElementById("divide-h")
-                .addEventListener("click", () =>  {
-                    if (this.activeG && this.activeG.activeW.activeP.sy >= 0.04)
+        const dH = document.getElementById("divide-h")
+        const dV = document.getElementById("divide-v")
+        dH.addEventListener("click", () =>  {
+                    if (this.activeG && !dH.classList.contains("off"))
```
Comment (daonb) [inline src/terminal7.js:162]: cut gestures still work... You can copy the logic there or maybe test it in `Pane.split`?

**Comment by daonb** [comment]
I've ended up taking this out as the tests are failing.

**Comment by eyal-sasson** [comment]
Should I fix the tests?

**Comment by eyal-sasson** [comment]
Is there any way to reopen this pr? Or will I have to make another one?

**Pull request merged**